src/schemas/json/cryproj.52.schema.json

```diff
@@ -2227,7 +2227,7 @@
 				"r_ShadersRemoteCompiler",
 				"r_ShadersSubmitRequestline",
 				"r_ShadowBlur",
-				"r_ShadowBluriness",
+				"r_ShadowBlurriness",
```

Comment:
This was officially misspelled, but has been [removed entirely](https://docs.cryengine.com/display/RN/CRYENGINE+5.5.0+Preview+5)